### PR TITLE
feat: add setting page

### PR DESCRIPTION
**Pull request by pelzman**
**Description of PR**

added Settings page
made the switch a reusable component without using external library
Link [CU-86bwt2t67](https://app.clickup.com/t/86bwt2t67)
---

Please check all that apply.

#### Checklist:

- [x] Rebased with master
- [x] PR Description provided
- [ ] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [x] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [ ] Does the endpoint return the correct success/error format?

---

**Screenshots**  

---

**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

File: apps/client/src/pages/Dashboard/Settings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  {
+    section: 'Communication Settings',
+    items: [
+      {
+        label: 'Email Notifications',
+        description: 'Receive Important Updates and Notifications via email',
+        accessor: true,
+      },
+    ],
+  },
```
Comment (KristofaJosh) [inline apps/client/src/pages/Dashboard/Settings/index.tsx:15]: let's add **disabled** for each item...

I see a possibility where backend can disable and option for a user

Comment (pelzman) [reply apps/client/src/pages/Dashboard/Settings/index.tsx:29]: oh i didnt know you dropped a comment here oo boss, once you review this i will add it

File: apps/client/src/components/elements/switcher/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+//       <input
+//         name="toggle"
+//         ref={inputRef}
+//         className={styles.switcherWrapperInput}
+//         checked={checked}
+//         type="checkbox"
+//       />
+//     </div>
+//   )
+// }
```
Comment (KristofaJosh) [inline apps/client/src/components/elements/switcher/index.tsx:44]: remove the comments

**Review by KristofaJosh — APPROVED** [review]
Remove unused code/commented code

LGTM!

**Review by Jidekas — APPROVED** [review]

**Review by AnosikeChidinma — APPROVED** [review]

**Pull request merged**